/**:
  ros__parameters:
    z_offset: 0.05
    yaw_offset: -1.5715
    x_world_offset: 0.60
    y_world_offset: -0.25
    z_world_offset: 0.0
